crates/sui-graphql-rpc/src/extensions/logger.rs

```diff
@@ -141,29 +137,43 @@ impl Extension for LoggerExtension {
                         }
                     }
 
-                    error!(
-                        target: "async-graphql",
-                        "[Response] path={} message={}", path, err.message,
-                    );
-                } else {
-                    error!(
-                        target: "async-graphql",
-                        "[Response] message={}", err.message,
-                    );
+                    if let Some(async_graphql_value::ConstValue::String(val)) = error_code {
```

Comment:
nit: Call this `error_code`, and then you can also turn `error_code = val` into just `error_code` below?